src/StripeTests/Entities/StripeBalanceTransactionTest.cs

```diff
@@ -0,0 +1,95 @@
+namespace StripeTests
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using System.Text;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class StripeBalanceTransactionTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            string json = GetFixture("/v1/balance/history/txn_123");
+            var balanceTransaction = Mapper<StripeBalanceTransaction>.MapFromJson(json);
+            Assert.NotNull(balanceTransaction);
+            Assert.IsType<StripeBalanceTransaction>(balanceTransaction);
+            Assert.NotNull(balanceTransaction.Id);
+            Assert.Equal("balance_transaction", balanceTransaction.Object);
+        }
+
+        [Fact]
+        public void DeserializeWithExpansions()
+        {
+            // We test all balance transaction possible source types to ensure the deserializer
+            // works as expectes.
+            var json = GetResourceAsString("api_fixtures.balance_transaction_with_expansion.json");
+            var balanceTransactions = Mapper<StripeList<StripeBalanceTransaction>>.MapFromJson(json);
+
```

Comment:
I would just add a couple of asserts here:
```c#
Assert.NotNull(balanceTransactions.Data)
Assert.IsTrue(balanceTransactions.Data.Count >= 9)
```
just to avoid exceptions when trying to access something that doesn't exist if the deserialization goes wrong (or if something is wrong with the fixture itself).

Reply:
@ob-stripe done! PTAL